whylog/tests/test_constraints/__init__.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from whylog.constraints import IdenticalConstraint
+from whylog.constraints.exceptions import ConstraintVerificationError
```

Comment:
Because there is nothing important to test. I will write test soon after implementing verify method. I am still not sure about it.